core/tests/protractor_desktop/topicsAndSkillsDashboard.js

```diff
@@ -55,12 +52,44 @@ describe('Topics and skills dashboard functionality', function() {
     topicsAndSkillsDashboardPage.get();
   });
 
-  it('should add a new topic to list', function() {
+  it('should add a new topic to list and delete it', function() {
     topicsAndSkillsDashboardPage.expectNumberOfTopicsToBe(0);
-    topicsAndSkillsDashboardPage.createTopic('Topic 1', true);
+    topicsAndSkillsDashboardPage.createTopic(
+      'Topic 1', 'Topic 1 description', 'Mathematics', true);
 
     topicsAndSkillsDashboardPage.get();
     topicsAndSkillsDashboardPage.expectNumberOfTopicsToBe(1);
+    topicsAndSkillsDashboardPage.deleteTopicWithIndex(0);
+    topicsAndSkillsDashboardPage.expectNumberOfTopicsToBe(0);
+  });
+
+  it('should filter the topics', function() {
+    topicsAndSkillsDashboardPage.expectNumberOfTopicsToBe(0);
+    topicsAndSkillsDashboardPage.createTopic(
+      'Alpha', 'Alpha description', 'Mathematics', true);
+    topicsAndSkillsDashboardPage.get();
+    topicsAndSkillsDashboardPage.createTopic(
+      'Beta', 'Beta description', 'Mathematics', true);
+
+    topicsAndSkillsDashboardPage.get();
+    topicsAndSkillsDashboardPage.expectNumberOfTopicsToBe(2);
+
+    topicsAndSkillsDashboardPage.filterTopicsByKeyword('alp');
+    topicsAndSkillsDashboardPage.expectNumberOfTopicsToBe(1);
+    topicsAndSkillsDashboardPage.resetTopicFilters();
+
+    topicsAndSkillsDashboardPage.filterTopicsByKeyword('be');
```

Comment:
Assert, before this, that the number of topics is 2.

Ditto for all other actions: assert the "before" status as well as the "after" status.

Reply:
Updated this. Done